packages/hierarchies/src/hierarchies/NodeSelectQueryFactory.ts

```diff
@@ -475,21 +507,46 @@ function createLabelGroupingBaseParamsSelectors(byLabel: ECSqlSelectClauseLabelG
   return selectors;
 }
 
-function createPropertyGroupSelectors(propertyGroup: ECSqlSelectClausePropertyGroup) {
+async function createPropertyGroupSelectors(
+  propertyGroup: ECSqlSelectClausePropertyGroup,
+  fullClassName: string,
+  classLoader: (fullName: string) => Promise<EC.Class>,
```

Comment:
what's the point to pass class name + a function that loads the class? maybe just load it beforehand and pass it here?

Reply:
Changed, now the class is being provided